index.js

```diff
@@ -540,4 +540,14 @@ DumbPipe.registerOpener("reload", function(message, sender) {
 
 DumbPipe.registerOpener("exit", function(message, sender) {
   window.close();
-});
\ No newline at end of file
+});
+
+DumbPipe.registerOpener("backgroundCheck", function(message, sender) {
+  if (navigator.mozAlarms) {
+    navigator.mozAlarms.add(new Date(Date.now() + 10000), 'ignoreTimezone', {});
+
+    navigator.mozSetMessageHandler('alarm', function() {
+      navigator.mozAlarms.add(new Date(Date.now() + 10000), 'ignoreTimezone', {});
```

Comment:
Nit: would be worth specifying the timeout in a variable.

Also, any thoughts on why you picked 10 seconds? It sounds a bit fast to me. In any case, it'd be good to include a comment explaining the reasoning here.